data/json/mapgen/helipad.json

```
@@ -122,8 +122,8 @@
         { "vehicle": "forklift", "x": 29, "y": 6, "chance": 95, "rotation": 180, "status": 1 },
         { "vehicle": "parkinglotbasic", "x": 4, "y": 9, "chance": 75, "rotation": 270, "status": -1 },
         { "vehicle": "flatbed_truck", "x": 11, "y": 4, "chance": 25, "rotation": 270, "status": -1 },
-        { "vehicle": "SmallFlier3_Wreck", "x": 9, "y": 38, "chance": 75, "rotation": 90, "status": -1 },
-        { "vehicle": "2seater2", "x": 24, "y": 33, "chance": 25, "rotation": 270, "status": 1 },
+        { "vehicle": "Medevac1", "x": 9, "y": 38, "chance": 1, "rotation": 90, "status": -1 },
+        { "vehicle": "helicopter_apache_1a", "x": 24, "y": 1, "chance": 25, "rotation": 270, "status": 1 },
```

```suggestion
        { "vehicle": "helicopter_apache_1a", "x": 24, "y": 33, "chance": 1, "rotation": 270, "status": 1 },
```
Did you mean to do this?

---

Yes indeed!